packages/cli/src/utils/local.js

```diff
@@ -228,7 +228,10 @@ const loadAppRawUsingImport = async (
   try {
     // zapierwrapper.mjs is only available since zapier-platform-core v17.
     // And only zapierwrapper.mjs exposes appRaw just for this use case.
-    appRaw = (await import(wrapperPath)).appRaw;
+    // Convert path to file:// URL for Windows compatibility with ESM imports
+    const pathForUrl = path.resolve(wrapperPath).replace(/\\/g, '/');
+    const wrapperUrl = new URL(`file:///${pathForUrl.replace(/^\//, '')}`);
```

Comment:
I could be wrong but should `file:///` be `file://` ? ( just judging from the comment above )

Reply:
Looks like it's part of the [File URI spec](https://en.wikipedia.org/wiki/File_URI_scheme), the format is `file://host/path` but if `host` is empty it means it's your local machine - good question, TIL!

Reply:
Ahh thank you!